File: com.vorwardit.wordsworthcms/Models/PageLayout.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace com.vorwardit.wordsworthcms.Models
{
    public class PageLayout
    {
        public Guid PageLayoutId { get; set; }

        public Guid LayoutId { get; set; }

        [ForeignKey("LayoutId")]
        public Layout Layout { get; set; }

        [MaxLength(50)]
        public string Name { get; set; }

        public string Body { get; set; }
    }
}

File: com.vorwardit.wordsworthcms/lib/admin/Domain/PageLayout.ts
module app.domain {
    export interface IPageLayout {
        pageLayoutId: string;
        layoutId: string;
        name: string;
        body: string;
    }

    export class PageLayout implements IPageLayout {
        constructor(
            public pageLayoutId: string,
            public layoutId: string,
            public name: string,
            public body: string
        ) {
        }
    }
}